src/tests/JIT/superpmi/superpmicollect.cs

```diff
@@ -606,7 +603,7 @@ private static void Usage()
             Console.WriteLine("If -mch is not given, all generated files are deleted, and the result is simply the exit code");
             Console.WriteLine("indicating whether the collection succeeded. This is useful as a test.");
             Console.WriteLine("");
-            Console.WriteLine("If the COMPlus_AltJit variable is already set, it is assumed SuperPMI collection is already happening,");
+            Console.WriteLine("If the COMPlus_JitName variable is already set, it is assumed SuperPMI collection is already happening,");
```

Comment:
I imagine this won't change the way we do in jitutils like dasm or pmi? https://github.com/dotnet/jitutils/search?q=altjit

Reply:
That's right; jit-diff with -altjit specified still uses the altjit variables.